### ISSUE TITLE
Option to Disable AI?

### ISSUE THREAD

**Issue by Crjustus** [excalidraw/excalidraw#8671]
I work for a school district that restricts the use of AI in instructional technology tools online. Is there an option to disable the AI functionality of Excalidraw? If there is not an option, is that something that could be added to the service?

**Comment by marcellodomenis** [comment]
You can already disable the AI functionality with:

```
const options = { aiEnabled: false };
```

![Image](https://github.com/user-attachments/assets/c2bae280-e660-412e-8f11-585885efb682)

@Mrazator I'm not sure if you'd like to implement something more accessible in the UI to disable the AI functionality, but this could also make it easier for students to toggle the features back on, right?

**Comment by Mrazator** [comment]
Yeah, though I assumed the question was about the UI, as this prop isn't exposed to be turned on / off from the end user perspective.

**Comment by Mrazator** [comment]
To elaborate further, as of now, it's already exposed in https://plus.excalidraw.com/.
![Image](https://github.com/user-attachments/assets/f71e3674-78f4-47cb-a743-91568bcfc373)

**Comment by JaredWyns** [comment]
Here's to hoping this is more easily exposed in non-workspace version the future. Seeing that as a non-removable option in the menu as a would-be new user (discovered via obsidian plugins) is an immediate turn-away. I'll revisit examining excalidraw when this is in place.

**Comment by Sparky1492** [comment]
Is/Has there been any update/progress for this?

@chari-md , where would this setting be set?
> You can already disable the AI functionality with:
> 
> ```
> const options = { aiEnabled: false };
> ```



I would like to have the option to disable AI tools.  I feel AI is getting shoved down our throats with now option to not partake if we do now want to.  
I understand it might not be sending anything (as of now) not being configured, there no re-assurance of this.
I have looked and searched and could not find a way to disable AI within the plugin.

When the option is enabled the AI options would be gone

![Image](https://github.com/user-attachments/assets/9f9ee98c-c96a-4d38-8afb-54e00b1e9922)


If you need/want additional information or need me to do anything, please let me know.
Thank you for your time.

**Comment by dwelle** [comment]
Hi.

The Obsidian-excalidraw plugin isn't maintained by us so we can't comment on that.

As for Excalidraw+, as mentioned, you can disable for the whole workspace.

On excalidraw.com (the free app), there are no immediate plans to make the UI configurable in this way as as it wouldn't have much benefit. All AI features are labeled as such so if you don't want to use it, you don't have to.

I'm closing this for now as non-actionable.
